src/common-imports.ts

```diff
@@ -19,6 +20,7 @@ export default function registerCustomComponents(extra = {}): Record<string, any
     ...commonComponents,
     ...interactionDesignerComponents,
     ...blockEditorsComponents,
+    ...blockEditorsChoicesComponents,
```

Comment:
don't forget to do the same in `lib.ts` 😉
(alternatively, you can add `export * from './choices'` to `/block-editors/index.ts` and you won't need either)

Reply:
Good catch, using `export * from './choices'`